Sources/Core/Subject/Subject.swift

```diff
@@ -24,9 +24,9 @@ import Foundation
 /// @class Subject
 /// This class is used to access and persist user information, it represents the current user being tracked.
 class Subject : NSObject {
-    private var standardDict = Payload()
+    private var standardDict: [String : String] = [:]
     private var platformContextManager = PlatformContext()
-    private var geoLocationDict: [String : NSObject] = [:]
+    private var geoDict: [String : NSObject] = [:]
```

Comment:
NSObject
There are some NSNumber in this file too, should they also be changed?

Reply:
I think we need to keep using the NSObject and NSNumber for the geolocation in Subject because some of the properties are nullable integers and numbers and Objective-C doesn't support nullable integers, only using NSNumber. So for compatibility with Obj-C we have to keep them like this.